tests/e2e/tests/bookinfo/demo_test.go

```diff
@@ -450,6 +450,22 @@ func TestDbRoutingMysql(t *testing.T) {
 		fmt.Sprintf("Success! Response matches with expected! %s", respExpr), t)
 }
 
+func TestVMExtendsIstio(t *testing.T) {
```

Comment:
can you add the rest, ie the mariadb install and the routing rules etc to use the mysql db
https://istio.io/docs/guides/integrating-vms.html#using-the-mysql-service

Reply:
@ldemailly - I think that s for you guys to do. This is a template for getting a VM test running.

Reply:
k, let me know when it's ready to hand off and passing minimally then
(though applying 2 yaml should be somewhat easy)